src/bootstrap/defaults/config.library.toml

```diff
@@ -10,6 +10,6 @@ bench-stage = 0
 incremental = true
 
 [llvm]
-# Will download LLVM from CI if available on your platform (Linux only for now)
+# Will download LLVM from CI if available on your platform.
 # https://github.com/rust-lang/rust/issues/77084 tracks support for more platforms
```

Comment:
Hmm, I'm not sure it makes sense to keep this comment now the issue is closed?

Reply:
you're right, i forgot to delete that one. one sec